src/template.js

```diff
@@ -384,6 +395,22 @@ require("../html5-common/js/utils/constants.js");
       this.controller.notify(this.controller.EVENTS.FULLSCREEN_CHANGED,
                              { "isFullScreen" : false, "paused" : event.target.paused });
     };
+
+    // Plugin must support the BITRATE_CONTROL feature notify the controller of this event.
```

Comment:
I don't know if i agree with this - why not report it even if you can't select one?